wrapper.go

```diff
@@ -30,32 +30,32 @@ const (
 )
 
 // KeyType defines types of cryptographic keys.
-type KeyType string
+type KeyType int
```

Comment:
One last comment around these consts, let's use `uint` or `uint32`  for them since they can't really be negative values.

Reply:
Done in ccf83ba 👍